### PR TITLE
Always return created obj back on REST calls

### PR DESCRIPTION
**Pull request by sulochan**
- we were not returning the created objects back on
CREATE calls on rest api. This patch fixes that.

### CODE REVIEW

**Comment by jimbaker** [comment]
The changes look good, but we should have some verifying tests in `test_api.py` - everything is set up as far as I can tell for doing such tests.

**Comment by sulochan** [comment]
Added tests. Is good for review and merge.

File: craton/inventory/api/v1/hosts.py [hunk 1]
```diff
@@ -127,11 +126,9 @@ def delete(self, id):
         # NOTE(sulo): this is not that great. Find a better way to do this.
         # We can pass multiple keys suchs as key1=one key2=two etc. but not
```
Comment (mwturvey) [inline craton/inventory/api/v1/hosts.py]: Is this comment still valid?

**Comment by mwturvey** [comment]
+1

File: craton/inventory/api/v1/hosts.py (hunk 1 above)
Comment (sulochan) [inline craton/inventory/api/v1/hosts.py]: yep, still valid.

**Comment by ChristopherMSpencer** [comment]
+1 fixed my issues on create region

**Comment by jimbaker** [comment]
LGTM

**Pull request merged**